google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/Row.java

```diff
@@ -167,4 +171,30 @@ private int getLast(@Nonnull String family, @Nullable ByteString qualifier, int
     }
     return index;
   }
+
+  /** wraps the protobuf {@link com.google.bigtable.v2.Row} */
+  public static Row fromProto(@Nonnull com.google.bigtable.v2.Row rowProto) {
+    Preconditions.checkArgument(rowProto != null, "Row must not be null");
+
+    DefaultRowAdapter adapter = new DefaultRowAdapter();
+    RowBuilder<Row> rowBuilder = adapter.createRowBuilder();
+    rowBuilder.startRow(rowProto.getKey());
+
+    for (Family family : rowProto.getFamiliesList()) {
+      for (Column column : family.getColumnsList()) {
+        for (Cell cell : column.getCellsList()) {
+          rowBuilder.startCell(
+              family.getName(),
+              column.getQualifier(),
+              cell.getTimestampMicros(),
+              cell.getLabelsList(),
+              column.getCellsCount());
```

Comment:
@igorbernstein2, what's the right value for "size" here?  I can't imagine that `column.getCellsCount()` is right.

Reply:
My apologies for adding `cellsCount` instead of `cell#getValue`'s `size`.

updated the same.